### DISCUSSION TITLE
Why declare routes at the test level?

### DISCUSSION THREAD

**Discussion by mccataldo** [cypress-io/cypress-realworld-app#730] [Q&A]
In the real world, it seems like a lot of work to declare routes at the test level, so I wonder why that strategy is maintained in this project. For example, if we create a new test, we're guaranteed to have to create aliased routes whereas if we declare these routes in a global pre-hook, it eliminates that hassle as the app scales.

**Comment by amirrustam** [comment]
Great question. This is a preferential practice but also can vary based on the test suite. Can you share a specific spec you are referring to within the RWA?

**Comment by mccataldo** [comment]
Thanks @amirrustam. In the spec, `cypress/tests/ui/notifications.spec.ts`, the following aliased routes are declared:
```
    cy.route("GET", "/notifications").as("getNotifications");
    cy.route("POST", "/transactions").as("createTransaction");
    cy.route("PATCH", "/notifications/*").as("updateNotification");
    cy.route("POST", "/comments/*").as("postComment");
```
and in most other specs like `cypress/tests/ui/new-transaction.spec.ts` the same aliases like `getNotifications` are being created.
